src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/Formatting/TestFiles/Expected/FormatOnPaste.cshtml

```diff
@@ -48,13 +48,13 @@
                 @foreach (var item in ViewBag.RequestOrder)
 
                 </tbody>
-
+                
             </table>
         </div>
     </div>
     <div>
-        @if (true)
-        {
+    @if (true)
+    {
             <a>
```

Comment:
if we're changing things should indentation of the `<a>` change here?

Reply:
So this is an invalid document, and is an example of where the Razor syntax tree is not helping us. The close brace in for the `@if` is parsed as markup text, not C# text, either because of the unclosed `<a>` or the unclosed `<div>` on line 44. In other words, the `if` doesn't fix the indentation of the `<a>` because it's not a valid `if`.

Either way, we don't get a valid Html or C# document to format, so its a little bit "you get what you get and you don't get upset".

Reply:
ah got it. This is where PR context would have helped if I kept expanding in the diff viewer